internal/rest/avi_obj_vs.go

```diff
@@ -790,24 +810,44 @@ func (rest *RestOperations) AviVsCacheDel(rest_op *utils.RestOp, vsKey avicache.
 
 			switch vs_cache_obj.ServiceMetadataObj.ServiceMetadataMapping("VS") {
 			case lib.GatewayVS:
-				updateOptions := status.UpdateOptions{
-					ServiceMetadata: vs_cache_obj.ServiceMetadataObj,
-					Key:             key,
-					VSName:          vs_cache_obj.Name,
-					Tenant:          vs_cache_obj.Tenant,
-				}
-				statusOption := status.StatusOptions{
-					ObjType: lib.Gateway,
-					Op:      lib.DeleteStatus,
-					Key:     key,
-					Options: &updateOptions,
-				}
-				utils.AviLog.Infof("key: %s Publishing to status queue, options: %v", updateOptions.ServiceMetadata.Gateway, utils.Stringify(statusOption))
-				status.PublishToStatusQueue(updateOptions.ServiceMetadata.Gateway, statusOption)
-				// The pools would have service metadata for backend services, corresponding to which
-				// statuses need to be deleted.
-				for _, poolKey := range vs_cache_obj.PoolKeyCollection {
-					rest.DeletePoolIngressStatus(poolKey, true, vs_cache_obj.Name, key)
+				if vs_cache_obj.ParentVSRef.Name != "" {
```

Comment:
Similar to https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/1538/files#diff-741dc2fc35a7d33e3e1cc587ac277346ff1edf2a7cdbdfead7aadfba29bfd533R563 
vs_cache_obj should be vsCacheObj

Reply:
Done